frontend/src/common/queries/collections.ts

```diff
@@ -153,8 +153,8 @@ function fetchCollection() {
 
     const url = apiTemplateToUrl(API_URL + API.COLLECTION, { id });
 
-    let response = await fetch(url, DEFAULT_FETCH_OPTIONS);
-    let json = await response.json();
+    const response = await fetch(url, DEFAULT_FETCH_OPTIONS);
+    const json = await response.json();
```

Comment:
Updating `response` and `json` to `const` to help identify them as different objects from the published counterpart's response (`publishedCounterpartResponse`) and json (`publishedCounterpartJSON`) objects below.